test/diagnostics/Localization/en_localization.swift

```diff
@@ -1,12 +1,18 @@
-// RUN: %target-typecheck-verify-swift -localization-path %S/Inputs -locale en
+// RUN: not %target-swift-frontend -typecheck %s 2>&1 -localization-path %S/Inputs -locale en | %FileCheck %s --check-prefix=CHECK_NONAMES
+// RUN: not %target-swift-frontend -debug-diagnostic-names -localization-path %S/Inputs -locale en -typecheck %s 2>&1 | %FileCheck %s --check-prefix=CHECK_NAMES
```

Comment:
I changed the tests to `check` statements -like `debug-diagnostic-names.swift`, to be able to check on the names.

Reply:
I'd suggest we add a separate test file for that instead of modifying existing.